assets/model_monitoring/components/src/shared_utilities/constants.py

```diff
@@ -47,6 +47,19 @@
 UPPER_BOUND_PARAM = 'upperBound'
 VERSION_PARAM = 'version'
 
+# Keywords in metrics schema
+AGGREGATE = 'aggregate'
+GROUP = 'group'
+GROUP_PIVOT = 'group_pivot'
+METRIC_NAME = 'metric_name'
```

Comment:
Can we prepend/append some term to distinguish between random constants and column keys for signal metrics?
maybe SIGNAL_METRIC_GROUP_COLUMN or GROUP_COLUMN? etc..

Reply:
Prepend column keys with `SIGNAL_METRICS_`.